index.html

```diff
@@ -1,278 +0,0 @@
-                        <a href="#"><img src="arquivos/img-story/respondeai 1.svg" alt=""></a>
-                        <a href="#"><img src="arquivos/img-story/filomoderna 1.svg" alt=""></a>
-                        <a href="#"><img src="arquivos/img-story/memeriagourmet 1.svg" alt=""></a>
-                    </div>
-                    <div class="storys-name">
-                        <p class="name-1">9gag</p>
-                        <p class="name-2">meowed</p>
-                        <p class="name-3">barked</p>
-                        <p class="name-4">nathanwpyle...</p>
-                        <p class="name-5">wawawiwac...</p>
-                        <p class="name-6">respondeai</p>
-                        <p class="name-7">filomoderna</p>
-                        <p class="name-8">memeriago...</p>
-                    </div>
-                </div>
-
-
-                <div class="post">
-                    <div class="barra-post">
-                            <img src="arquivos/img-story/meowed 1.svg" alt="" class="logo-post">
-                            <p>meowed</p>
-                            <img src="arquivos/imgs/ellipsis-horizontal 1.svg" alt="" class="elipse">
-                    </div>
-
-                    <img src="arquivos/imgs/gato-telefone 1.jpg" alt="">
-
-                    <div class="barra-post-inf">
-                        <div class="barra-post-inf-top">
-                            <div class="icone-post-left">
-                                <ion-icon name="heart-outline"></ion-icon>
-                                <ion-icon name="chatbubble-outline"></ion-icon>
-                                <ion-icon name="paper-plane-outline"></ion-icon>
-                            </div>
-                            <div class="icone-post-rigth">
-                                <ion-icon name="bookmark-outline"></ion-icon>
-                            </div>
-                        </div>
-                        <div class="barra-post-inf-bot">
-                            <img src="arquivos/img-story/respondeai 1.svg" alt="">
-                            <p>Curtido por <span>respondeai</span> e <span> 101.523 pessoas</span></p>
-                            <div class="barra-post-coment">
-                                <p> <span>gba_geteco</span> Muito fofo ahhhh!</p>
-                                <p> HÁ 2 HORAS </p>
-                                
-                            </div>
-                        </div>
-                       
-                    </div>
-
-                    <div class="comentario">
-                        <div class="comentario-left">
-                            <ion-icon name="happy-outline"></ion-icon>
-                            <input type="text" placeholder="Adicione um comentário..." class="pesquisa no-mobile" />
-                          
-                            <p></p>
```

Comment:
Essa tag não ta sendo usada então não faz necessário ter ela no codigo, se o foco for espaçamento ou algo do tipo, é melhor tratar no css utilizando margin.